package.json

```
@@ -52,7 +52,8 @@
     "deploy": "gh-pages -d dist",
     "preprintable": "npm run clean-printable",
     "printable": "node ./src/scripts/concatenate-docs.js",
-    "jest": "jest"
+    "jest": "jest",
+    "cypress:open": "cypress open"
```

```suggestion
    "cypress:open": "cypress run"
```
I think it has to be `run` as `open` opens a visual browser instead of headless and for CI use (which is I think will be an option for future if this repo is having tests) it is better as headless.

Although I am not completely sure about it. let me know 👍

---

Sometimes we need visual browser for debugging :) Maybe we can add another `script` like `cypress:run` for CI usage?

---

> Sometimes we need visual browser for debugging :) Maybe we can add another script like cypress:run for CI usage?

yes, both should be there if we are going to setup CI. for this PR , the current seems fine as I think adding CI might be in a separate PR. lets hear from others.